### ISSUE TITLE
New TestStep: SweepFromFile

### ISSUE THREAD

**Issue by alnlarsen** [opentap/opentap#995]
This is related to #786 

After discussing the feature request with @jfaaborg and @AsgerIversen , we agreed
that this is a good feature, but we don't want to add this functionality on top of
the existing `Sweep Step`. Instead, we want to create a new step, `Sweep From File`.

Here is a mock-up of the new step:
![MicrosoftTeams-image](https://user-images.githubusercontent.com/95965749/215769334-703ff9f6-06b4-4d22-80ac-9755193d28e4.png)

We agreed to do it this way for a couple of reasons:

#### We don't want to exclusively support CSV. 
Plugins that implement `ITableImport` can declare what file extensions they support. `Sweep From File` will accept any file format as long as there is a plugin installed which is capable of handling that format.

#### Mapping columns to parameters
When the user is manually adding rows to a table within the editor, it is easy
to manage the mapping between the headers in the table, and the names of the 
parameterized parameters because the table is managed by OpenTAP itself.

Manually created sweep grids are supposed to be small enough to be manually editable in
the test plan settings editor. That's why there is an edit button. The same assumption
cannot be made about file content.

We cannot make that assumption when the data is potentially from an external source.
We agreed that the user should be able to configure a mapping between columns in the file,
and the parameters that are actually parameterized. 

Furthermore, when using an external source, it is highly likely that the source may have
columns that are not parameterized on the step that uses them. 

In short, the user should be able to see what columns are available in the file, and they should be able to map each column to a parameter.

**Comment by rmadsen-ks** [comment]
Although this already has an implementation we will wait with this until 9.23.

**Comment by rmadsen-ks** [comment]
Lets call this Sweep from CSV and move it to the CSV plugin.
